scipy/linalg/_decomp.py

```diff
@@ -336,18 +338,26 @@ def eigh(a, b=None, lower=True, eigvals_only=False, overwrite_a=False,
         Whether to check that the input matrices contain only finite numbers.
         Disabling may give a performance gain, but may result in problems
         (crashes, non-termination) if the inputs do contain infinities or NaNs.
-    turbo : bool, optional
-        *Deprecated since v1.5.0, use ``driver=gvd`` keyword instead*.
+    turbo : bool, optional, deprecated
         Use divide and conquer algorithm (faster but expensive in memory, only
```

Comment:
Let's get rid of the text here completely because it is not offering anything of substance and leave just the warning message.